src/services/auth/error/cognitoError.ts

```diff
@@ -5,3 +5,21 @@ export class CognitoError extends Error {
 		this.code = code;
 	}
 }
+
+export class BadRequestException {
+	public message: string;
+	public status: number;
+	constructor(message: string) {
+		this.status = 400;
+		this.message = message;
+	}
+}
+
+export class InternalServerErrorException {
+	public message: string;
+	public status: number;
+	constructor(message: string) {
+		this.status = 500;
+		this.message = message;
+	}
+}
```

Comment:
let's create a function for these things, instead of classes, and let's create the type of the function.
For example, if it is going to be an object, that returns 
`{
statusCode: number,
message: string 
}`